### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yurykopytov**

### CODE REVIEW

**Comment by yurykopytov** [comment]
Add Yury Kopytov to the student list

File: README.md [hunk 1]
```diff
@@ -10,3 +10,4 @@
 | 0 | Ава Катушка     | Три идиота                             |
 | 1 | Наташа Смирнова | Меланхолия                             |
 | 2 | Веня Стаховский | Интерстеллар                           |
+| 3 | Юрий Копытов    | Просто кровь                           |
```
Comment (yurykopytov) [inline README.md]: Add Yury Kopytov to the student list

**Comment by ava8katushka** [comment]
Please resolve the conflicts and update the name of the pull request to be "Add Your Name to the student list"

**Comment by ava8katushka** [comment]
All looks good, but update the name of the pull request from Update README.md to Add Yury Kopytov to the student list

File: README.md [hunk 2]
```diff
@@ -10,4 +10,6 @@
 | 0 | Ава Катушка     | Три идиота                             |
 | 1 | Наташа Смирнова | Меланхолия                             |
 | 2 | Веня Стаховский | Интерстеллар                           |
-| 3 | Михаил Антаманов| Питер-ФМ                               |
\ No newline at end of file
+| 3 | Михаил Антаманов| Питер-ФМ                               |
+| 4 | Юрий Копытов    | Просто кровь                           |
+
```
Comment (ava8katushka) [inline README.md]: Please remove the empty line.

**Comment by yurykopytov** [comment]
Add Yury Kopytov to the student list

**Comment by yurykopytov** [comment]
Add Yury Kopytov to the student list

**Review by yurykopytov — COMMENTED** [review]
Add Yury Kopytov to the student list

**Comment by ava8katushka** [comment]
> Add Yury Kopytov to the student list

Please, add this name as a name for your pull request. 
Now the name of your pull request is "Update README.md".
And resolve the conflicts.

**Pull request closed without merging**